internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1004,6 +1004,48 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"storage_blob_driver": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"storage_disk_driver": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"enabled": {
+							Type:     pluginsdk.TypeBool,
+							Optional: true,
+							Default:  true,
```

Comment:
We're moving away from exposing the `Enabled` property to users, particularly for AKS a fair amount of refactoring went in to remove this field for enabling/disabling features within the resource.

Could this be flattened into two separate properties instead
`disk_driver_enabled` and `disk_driver_version`